README.md

```diff
@@ -32,141 +38,63 @@ Here is my another open-source library :) https://github.com/fzyzcjy/flutter_smo
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/jsonmona"><img src="https://avatars.githubusercontent.com/u/105187344?v=4?s=100" width="100px;" alt="jsonmona"/><br /><sub><b>jsonmona</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=jsonmona" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/utilForever"><img src="https://avatars.githubusercontent.com/u/5622661?v=4?s=100" width="100px;" alt="Chris Ohk"/><br /><sub><b>Chris Ohk</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=utilForever" title="Documentation">📖</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Demezy"><img src="https://avatars.githubusercontent.com/u/38487319?v=4?s=100" width="100px;" alt="._."/><br /><sub><b>._.</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=Demezy" title="Documentation">📖</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/elliotsayes"><img src="https://avatars.githubusercontent.com/u/7699058?v=4?s=100" width="100px;" alt="elliotsayes"/><br /><sub><b>elliotsayes</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=elliotsayes" title="Documentation">📖</a></td>
-    </tr>
-    <tr>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/temeddix"><img src="https://avatars.githubusercontent.com/u/66480156?v=4?s=100" width="100px;" alt="Kim Dong-Hyun"/><br /><sub><b>Kim Dong-Hyun</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=temeddix" title="Code">💻</a> <a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=temeddix" title="Documentation">📖</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://github.com/Voklen"><img src="https://avatars.githubusercontent.com/u/56766748?v=4?s=100" width="100px;" alt="Alex Gorichev"/><br /><sub><b>Alex Gorichev</b></sub></a><br /><a href="https://github.com/fzyzcjy/flutter_rust_bridge/commits?author=Voklen" title="Documentation">📖</a></td>
-    </tr>
-  </tbody>
-</table>
-
-<!-- markdownlint-restore -->
-<!-- prettier-ignore-end -->
-
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+Thanks goes to these wonderful people!
+
+[![All Contributors](https://img.shields.io/github/all-contributors/fzyzcjy/flutter_rust_bridge?color=ee8449)](#contributors-)
+
+<a href="https://github.com/fzyzcjy/flutter_rust_bridge/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=fzyzcjy/flutter_rust_bridge" />
```

Comment:
Another issue is that the ordering cannot be customized, thus we lack some flexibilities IMHO. For example, we know @Desdaemon has done a lot of great job in this repo, so seems that we should put him to the second place. (I have not order all other contributors yet, but the all-contributors approach will have this flexibility)

Reply:
With this approach, the order of the badges follows the number of commits. The reason @Desdaemon is not in the second is because there were other contributors using small-sized, large number of commits in the PR. If the commits were squashed per PR, @Desdaemon actually would be the second I believe.

IMO the order would not that important, because the first purpose of README is mainly introducing the package, rather than all the contributors. GitHub's `Insights` tab is exactly for that :)

<img width="933" alt="image" src="https://user-images.githubusercontent.com/66480156/220843613-1c966495-719e-488e-b821-6a9c8dc8b51a.png">

Reply:
I appreciate your giving credits to main contributors. So, this would be a matter of choice:
- between having control over the order of contributions table even though it is misaligned
- Use orgnized contribution table, using the `Insignts` tab for detailed contribution view